Formula/pulumi.rb

```diff
@@ -2,8 +2,8 @@ class Pulumi < Formula
   desc "Cloud native development platform"
   homepage "https://pulumi.io/"
   url "https://github.com/pulumi/pulumi.git",
-      tag:      "v2.13.2",
-      revision: "8b8b538d08ee38fd4a50d67f3af50e917f6411ab"
+      tag:      "2.14.0",
```

Comment:
```suggestion
      tag:      "v2.14.0",
```

Upstream tag has a `v` before the umber. This should fix this CI issue.